browser/js/my-account/my-account.html

```diff
@@ -0,0 +1,79 @@
+<section id="account-page">
+	<div class="container">
+		<div class="row">
+			<div class="medium-4 large-4 columns">
+				<img class="thumbnail" height="250" width="250" src="http://dx3x.com/bills-progress/images/bill61_400x400.jpg">
+			</div>
+			<div class="medium-8 large-8 columns">
+				<div class="topOfPage">
+					<h3>{{ userSess.user.email }}</h3>
+					<h5>Your Account Info</h5>
+				</div>
+				<div class="row">
+					<div class="medium-6 large-12 columns">	
+						<p>Name: <input type="text" class="form-control" value="{{ userSess.user.fullName }}"></p>
+						<p>Address: 
+							<p>Street Address: <input type="text" class="form-control" value="Street Address"></p>
+							<p>City: <input type="text" class="form-control" value="City"></p>
+							<p>State: <input type="text" class="form-control" value="State"></p>
+							<p>Zip Code: <input type="text" class="form-control" value="Zip Code"></p>
+						</p>
+					</div>
+				</div>
+
+				<div class="row">
+					<div>
+						<a class="button medium expanded">Save Changes</a>
+					</div>
```

Comment:
I'm wondering if we should make the name/address show up not as an input, and then give the user the option to make changes?  I did this on the order page so it should be fairly easy to implement.  Not necessarily a priority though.